Library/Homebrew/extend/fileutils.rb

```diff
@@ -55,6 +92,7 @@ def mkdir(name, &_block)
       end
     end
   end
+
```

Comment:
I'd prefer if the PR didn't introduce empty lines unrelated to the changed code.